serverless/autoscaling/serverless-autoscaling-scale-bounds.adoc

```diff
@@ -0,0 +1,18 @@
+[id="serverless-autoscaling-scale-bounds"]
+= Scale bounds
+include::modules/common-attributes.adoc[]
+include::modules/serverless-document-attributes.adoc[]
+:context: serverless-autoscaling-scale-bounds
+
+toc::[]
+
+You can set scale bounds for an application to help prevent cold starts or control computing costs. Scale bounds determine the minimum and maximum numbers of pods that can serve an application at any given time. The `minScale` and `maxScale` annotations can be used to configure scale bounds.
+
+[NOTE]
+====
+The following annotations apply for the full lifetime of a revision. Even when a revision is not referenced by any route, the minimal pod number specified by the `minScale` annotation is still provided. Non-routeable revisions may be garbage collected, which enables Knative to reclaim the resources.
```

Comment:
AFAIK this is no longer true. MinScale no longer applies to non-routable revisions I think.

Reply:
Thank you, I really wanted to remove this but wasn't sure if it was still relevant 😄